sormas-backend/src/main/java/de/symeda/sormas/backend/util/PatchHelper.java

```diff
@@ -0,0 +1,330 @@
+	 * the list it is patched as all the other objects.
+	 * If there is no element with that UUID in the list the new element is added. This method is specific to the project, For the object
+	 * which re not instances of HasUuid (dont have an uuid),
+	 * all the elements are replaced with the values form json
+	 * 
+	 * @param existingObject
+	 * @param existingObjectField
+	 * @param listElementClass
+	 * @param listElement
+	 * @param <T>
+	 * @throws JsonProcessingException
+	 */
+	private static <T extends HasUuid> T addOrReplaceListElement(
+		T existingObject,
+		Field existingObjectField,
+		Class listElementClass,
+		JsonNode listElement) {
+		if (HasUuid.class.isAssignableFrom(listElementClass) && listElement.hasNonNull(EntityDto.UUID)) {
+			T existingListElement = (T) getListElementByUuid(existingObject, existingObjectField, listElement.get(EntityDto.UUID).textValue());
+			if (existingListElement != null) {
+				postUpdate(listElement, existingListElement);
+				//tempNewElementList.add(existingListElement);
+				return existingListElement;
+			} else {
+				//addObjectToList(tempNewElementList, listElement, listElementClass);
+				return (T) getObjectFromJsonNode(listElement, listElementClass);
+			}
+
+		} else {
+			//addObjectToList(tempNewElementList, listElement, listElementClass);
+			return (T) getObjectFromJsonNode(listElement, listElementClass);
+		}
+	}
+
+	/**
+	 * Replace the field value of the existing with the value from the json
+	 * 
+	 * @param existingObject
+	 * @param existingField
+	 * @param fieldJson
+	 * @param <T>
+	 * @throws IllegalAccessException
+	 * @throws JsonProcessingException
+	 */
+	private static <T> void setFieldValue(@NotNull T existingObject, @NotNull Field existingField, JsonNode fieldJson) {
+		Object fieldValue = null;
+
+		fieldValue = getObjectFromJsonNode(fieldJson, existingField.getType());
+		try {
+			existingField.set(existingObject, fieldValue);
+		} catch (IllegalAccessException e) {
+			e.printStackTrace();
+			throw new RuntimeException(
+				"Cannot access field " + existingField.getName() + " on the object of type: " + existingObject.getClass().getName());
+		}
+
+	}
+
+	/**
+	 * Deserialized the json to the java object. The method treats the special case of enumeration and also the case when the json is null
+	 * 
+	 * @param fieldJson
+	 * @param objectClass
+	 * @return
+	 * @throws JsonProcessingException
+	 */
+	private static Object getObjectFromJsonNode(JsonNode fieldJson, @NotNull Class objectClass) {
+		Object object = null;
+		if (!fieldJson.isNull()) {
+
+			if (objectClass.isEnum()) {
+				object = Enum.valueOf((Class<Enum>) objectClass, fieldJson.textValue());
+			} else {
+				try {
+					object = objectMapper.treeToValue(fieldJson, objectClass);
+				} catch (JsonProcessingException e) {
+					e.printStackTrace();
+					throw new RuntimeException("Cannot covert JSon" + fieldJson.asText() + " to object of class: " + objectClass.getName());
```

Comment:
please use RuntimeException(String message, Throwable cause) instead of e.printStackTrace(); throw new RuntimeException